case-studies/babelstream/plot_babelstream_cascade.py

```diff
@@ -76,7 +76,7 @@
 # -----------------------
 
 fig = plt.figure(figsize=(6, 5))
-ax = p3.plot.cascade(effs) 
+p3.plot.cascade(effs)
```

Comment:
Thanks for pointing this out.  What happened here is that I forgot to update the case studies when I was updating the examples...  We _should_ be assigning the output to a variable, and then using that to save the figure.  The code as written only worked because `plt.savefig` was implicitly picking up the figure created inside `cascade`.

I've fixed this in 3c31dfa96da5442c2bb6a0ff03ac76ad0968bf26.